kong/plugins/request-transformer/access.lua

```diff
@@ -116,6 +125,16 @@ local function transform_headers(conf)
 end
 
 local function transform_querystrings(conf)
+  -- Rename querystring(s)
+  if conf.rename.querystring then
+    local querystring = req_get_uri_args()
+    for _, old_name, new_name in iter(conf.rename.querystring) do
+      local value = querystring[old_name]
+      querystring[new_name] = value
+      querystring[old_name] = nil
+    end
+  end
+
```

Comment:
you're missing `req_set_uri_args(querystring)` to store the new updated values in the request